overlord/snapstate/handlers.go

```diff
@@ -2912,7 +2912,7 @@ func (m *SnapManager) undoLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 	// the cleanup is performed by snapd from core;
 	// when reverting a subsequent snapd revision, the restart happens in
 	// undoLinkCurrentSnap() instead
```

Comment:
the comment probably needs changing too?

Reply:
Right, nice catch, rewrote this